### PR TITLE
update openid-client

### PR DESCRIPTION
**Pull request by paulius-valiunas**

### CODE REVIEW

**Review by grigasp — APPROVED** [review]

**Review by Animagne — APPROVED** [review]

**Review by bsteinbk — APPROVED** [review]

**Review by pmconne — APPROVED** [review]

**Comment by ColinKerr** [comment]
I'm a little concerned about moving to cpx2.  It doesn't have many weekly downloads and it's only claim to fame is that it updates some dependencies that have security issues.  Those security issues are all regular expression denial of service problems which is not an issue for a tool we use in our build/test scripts.

I think we are safer sticking with cpx because if some malicious code does get stuck in one of the two it will be found more quickly in cpx than cpx2.

**Comment by calebmshafer** [comment]
I agree @ColinKerr, we can just continue to suppress that specific vulnerability even though it's annoying...

@paulius-valiunas mind removing the cpp upgrade from this PR and just doing the openid-client?

File: full-stack-tests/presentation/package.json [hunk 1]
```diff
@@ -44,7 +44,6 @@
     "@types/chai-as-promised": "^7",
     "@types/chai-jest-snapshot": "^1.3.0",
     "@types/chai-subset": "1.3.1",
-    "@types/cpx": "1.5",
```
Comment (calebmshafer) [inline full-stack-tests/presentation/package.json:47]: I think you may still need this dep

Comment (paulius-valiunas) [reply full-stack-tests/presentation/package.json]: yeah, I just wanted to reduce the noise with the first commit so I could see which changelog files were still relevant. I got the rest in separate commits

**Pull request merged**